source/slang/core.meta.slang

```diff
@@ -1232,6 +1247,22 @@ for (int tt = 0; tt < kTypeCount; ++tt)
     }
 }
 
+sb << "\n\n";
+sb << "// Cast from vector<T,1> to T, which is a scalar type\n";
+
+for (int tt = 0; tt < kBaseTypeCount; ++tt)
+{
+    if(kBaseTypes[tt].tag == BaseType::Void) continue;
+
+    const char* tname = kBaseTypes[tt].name;
+
+    sb << "extension " << tname << "\n";
+    sb << "{\n";
+    sb << "    __implicit_conversion(200)\n";
```

Comment:
kConversionCost_*

Reply:
The cost can be lower since float1 should in most cases be considered identical to float.

Reply:
I will make it 10.